src/olympia/reviewers/migrations/0005_remove_code_manager_waffle_switch.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.12 on 2020-06-02 13:28
+
+from django.db import migrations
+
+
+def delete_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
```

Comment:
Wasn't it a `Flag`? If so, then use the `Flag` model instead of the `Switch` one. Waffle has different types: https://waffle.readthedocs.io/en/stable/types/index.html